loops-and-arrays.js

```diff
@@ -30,7 +32,13 @@ const numbers = [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
 
-
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i]%2===0){
+        console.log(numbers[i]*2)
+    } else {
+        console.log(numbers[i]*3)
+    }
+}
 
```

Comment:
Tip, probeer berekeningen niet in een print/log te doen. Voor de rest is je code heel netjes geschreven en duidelijk te lezen! Complimenten!